Mathlib/Data/Seq/Computation.lean

```diff
@@ -310,8 +310,7 @@ theorem eq_of_bisim (bisim : IsBisimulation R) {s₁ s₂} (r : s₁ ~ s₂) : s
         exact False.elim h
       · rw [destruct_pure, destruct_think] at h
         exact False.elim h
-      · simp at h
-        simp [*]
```

Comment:
Sure that's a golf, but please reassure me that your linter does not flag this as a non-terminal simp!

Reply:
I am still working on the internal mechanics of the linter.  The first `simp` was indeed flagged, but a newer version of the linter will hopefully be aware of the tactic immediately following a `simp` and will use this information to decide whether a `simp` is considered terminal or not.  So,
* I will not revert this change, 
* I agree with you that this should not have been flagged,
* I do think that the current version is more robust than what it was before, though!

:smile: